app/services/profiles/update.rb

```diff
@@ -12,7 +12,7 @@ def initialize(user, updated_attributes)
       @user = user
       @profile = user.profile
       @updated_profile_attributes = updated_attributes[:profile] || {}
-      @updated_user_attributes = updated_attributes[:user] || {}
+      @updated_user_attributes = updated_attributes[:user].to_h || {}
```

Comment:
@citizen428 I did test this change, but wanted to bring it to your attention that I moved the `to_h` from the `update_user_attributes` method to this line in the initialize. That way we could use  `@updated_user_attributes[:profile_image]` instead of `@updated_user_attributes["profile_image"]` which made the testing on profile images simpler.